physics.hpp

```diff
@@ -103,5 +104,17 @@ vec3 phy::PhysicsObj::TotalForce()
 	}
 	return ResultantForce;
 }
+void phy::PhysicsObj::CalcVelocity()
```

Comment:
The function won't be running once a second, it will be called multiple times in 1 second.
Take time interval as an argument and use that. Updating the coordinates? Won't it be easier to do everything in 1 function?

Reply:
is there any fixed interval you want?

Reply:
it'll get disorganised if we do everything with one function!

Reply:
I dont know how to add the time constraint to the function. How do i call it after a certain interval of time.
@adityachirania we'll add accelaration as a data member and we'll have a different function for the coordinates of COM